src/EnergyPlus/ReportCoilSelection.hh

```diff
@@ -374,17 +385,29 @@ public: // methods
 		std::string const & coilType, // idf input object class name of coil
 		Real64 const totalCoolingCap, // {W} coil cooling capacity
 		bool const isAutoSize, // true if value was autosized
-		int const dataPltSizNum, // plant sizing structure index
+		int const inletNodeNum, // coil chw inlet node num
+		int const outletNodeNum, // coil chw outlet node num
+		int const dataWaterLoopNum // plant loop structure index
+	);
+
+	void
+	setCoilWaterHeaterCapacityNodeNums(
+		std::string const & coilName, // user-defined name of the coil
+		std::string const & coilType, // idf input object class name of coil
+		Real64 const totalHeatingCap, // {W} coil Heating capacity
+		bool const isAutoSize, // true if value was autosized
+		int const inletNodeNum, // coil chw inlet node num
+		int const outletNodeNum, // coil chw outlet node num
 		int const dataWaterLoopNum // plant loop structure index
 	);
 
 	void
-	setCoilWaterHeaterCapacity(
+	setCoilWaterHeaterCapacityPltSizNum(
 		std::string const & coilName, // user-defined name of the coil
 		std::string const & coilType, // idf input object class name of coil
 		Real64 const totalHeatingCap, // {W} coil Heating capacity
 		bool const isAutoSize, // true if value was autosized
-		int const dataPltSizNum, // plant sizing structure index
+		int const DataPltSizNum, // plant sizing structure index
```

Comment:
Was there a reason for capitalizing this?

Reply:
Hmm, don't remember changing  that.